### ISSUE TITLE
LGY: fill out ops template for new external service documentation

### ISSUE THREAD

**Issue by RudyOnRails** [department-of-veterans-affairs/va.gov-team#34077]
## BACKGROUND:
The basics of all new services need to be documented so Ops can know what it is, what it does, who runs it etc.

https://depo-platform-documentation.scrollhelp.site/developer-docs/Template.1887567914.html

## Additional Info
Per jbritt1 - https://dsva.slack.com/archives/CJYRZK2HH/p1639489531454300?thread_ts=1639428629.441700&cid=CJYRZK2HH

## Acceptance Criteria
- [ ] The template is filled out and sent to Ops team.

**Comment by saderagsdale** [comment]
LGY (Loan Guarantee Service) is the upstream data service we're using for Cert. of Eligibility.

**Comment by saderagsdale** [comment]
@scottymeyers How is documentation going for this one?

@cohnjesse have we already done a team walkthrough for LGY?

**Comment by scottymeyers** [comment]
@saderagsdale @cohnjesse no real progress on this yet from me. Jerek had walked me through the frontend for this and CoE, but it would be really helpful to see an example of this documentation from another service, to properly understand how to approach this.

**Comment by saderagsdale** [comment]
@Mottie @jerekshoe are there other examples of documentation from similar services that we can use as a starting point? Or can you point me toward someone who might know?

**Comment by Mottie** [comment]
I've never had to deal with a third party service.

**Comment by jerekshoe** [comment]
Yeah this is something I don’t have any experience with either. Jesse or Michel might have more knowledge on how to proceed

**Comment by saderagsdale** [comment]
Action item:
- Ask Matt, maybe the platform team about previous documentation.

**Comment by RudyOnRails** [comment]
There is a link to examples in the description above...
